assets/stylesheets/environment/tools/mixins/_grid-layout.scss

```diff
@@ -26,12 +26,13 @@
 ///   div {
 ///     @include grid-column(1 / 2, $float: right);
 ///   }
-@mixin grid-column($width, $full-width: tablet, $float: left) {
+@mixin grid-column($width, $full-width: desktop, $float: left) {
   padding: 0 ($gutter / 2);
+  max-width: $measure;
 
   @include media($full-width) {
     float: $float;
-    width: percentage($width);
+    max-width: percentage($width);
```

Comment:
I'm happy with this if we add the [<IE8 mixin](https://github.com/nhsalpha/betahealth/blob/develop/assets/stylesheets/environment/tools/mixins/_conditionals.scss#L72-L78) to set the `width` property.